sos/report/plugins/openstack_barbican.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (C) 2019 Mirantis, Inc., Denis Egorenko <degorenko@mirantis.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.plugins import Plugin, DebianPlugin, UbuntuPlugin
+
+
+class OpenStackBarbican(Plugin, DebianPlugin, UbuntuPlugin):
+    """OpenStack Barbican Secure storage service"""
+    plugin_name = "openstack_barbican"
+    profiles = ('openstack', 'openstack_controller')
+
+    packages = ('barbican-common', 'barbican-keystone-listener', 'barbican-worker',)
+
+    requires_root = False
+
+    def setup(self):
+        self.add_copy_spec([
+            "/etc/barbican/",
+        ])
```

Comment:
No need to use a list for single-items. `add_copy_spec()` takes a str as well.

Reply:
Done